### PR TITLE
feat(query): return COPY INTO TABLE file status

### LINKED ISSUE #7730: feat: COPY INTO returns more status
**Summary**

Returns:
```
file: Name of source file and relative path to the file
status: LOADED/SKIPPED
rows_loaded: Load how many rows
```

```
file	status	rows_loaded
s3://etharchive.org/pub/blocks/blocks_13000001_13001000.parquet	LOADED	1,000		
```

### PR DESCRIPTION
**Pull request by BohuTANG**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

* Return file status after COPY INTO TABLE

Fixes #7730

### CODE REVIEW

File: src/query/service/src/interpreters/interpreter_copy_v2.rs [hunk 1]
```diff
@@ -263,61 +266,68 @@ impl CopyInterpreterV2 {
...
 
-            Err(may_error.as_ref().unwrap().clone())
-        });
+                Err(may_error.as_ref().unwrap().clone())
+            });
+        }
 
-        Ok(build_res)
+        // Copy status.
+        let schema = Arc::new(DataSchema::new(vec![DataField::new(
```
Comment (Xuanwo) [inline src/query/service/src/interpreters/interpreter_copy_v2.rs:324]: Please change `CopyPlanV2` into a `has_result` plan.

Comment (BohuTANG) [reply src/query/service/src/interpreters/interpreter_copy_v2.rs:329]: Thank you.
I am working on why it does not return the result, you saved me 😢

**Comment by Xuanwo** [comment]
Cc @BohuTANG Is this PR still going on?

**Comment by BohuTANG** [comment]
No, closed.

**Pull request closed without merging**